feat(curriculum): add block 2 dialogs and tasks of the B1 course of the English Curriculum (#57471)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/674619b6699a06d064c9717f.md

---
id: 674619b6699a06d064c9717f
title: Task 37
challengeType: 22
dashedName: task-37
---

<!-- (Audio) Lisa: Did anything else come up while you were debugging? -->

# --instructions--

Listen to the audio and complete the sentence below.

# --fillInTheBlank--

## --sentence--

`Did anything else BLANK up while you were BLANK?`

## --blanks--

`come`

### --feedback--

Followed by `up` in this phrase, it means to arise or appear unexpectedly.

---

`debugging`

### --feedback--

This word refers to identifying and fixing errors or issues in code. Use the `-ing` form.

# --explanation--

> The phrase `come up` means to arise, appear, or occur unexpectedly. In this context, Lisa is asking if any other issues arose while James was debugging. Example: `Something important came up during the meeting, and we had to address it immediately.` (meaning something occurred during the meeting that needed immediate attention)  
> 
> The term `debugging` refers to the process of identifying and resolving errors or issues in code. Example: `The team spent hours debugging the software before the release.` (meaning they spent a long time identifying and solving these errors)
> 
> Lisa’s question asks if any other unexpected issues could be seen during the process of fixing code errors.

Reviewer: ```suggestion
`Come up` means to arise, appear, or occur unexpectedly. For example: 

`Something important came up during the meeting, and we had to address it immediately.` - Meaning something occurred during the meeting that needed immediate attention.  

`Debugging` refers to the process of identifying and resolving errors or issues in code. For example:

 `The team spent hours debugging the software before the release.` - Meaning they spent a long time identifying and solving these errors.

Lisa’s question asks if any other unexpected issues could be seen during the process of fixing code errors.
```

---
id: 674619b6699a06d064c9717f
title: Task 37
challengeType: 22
dashedName: task-37
---

<!-- (Audio) Lisa: Did anything else come up while you were debugging? -->

# --instructions--

Listen to the audio and complete the sentence below.

# --fillInTheBlank--

## --sentence--

`Did anything else BLANK up while you were BLANK?`

## --blanks--

`come`

### --feedback--

Followed by `up` in this phrase, it means to arise or appear unexpectedly.

---

`debugging`

### --feedback--

This word refers to identifying and fixing errors or issues in code. Use the `-ing` form.

# --explanation--

`Come up` means to arise, appear, or occur unexpectedly. For example: 

`Something important came up during the meeting, and we had to address it immediately.` - Meaning something occurred during the meeting that needed immediate attention.  

`Debugging` refers to the process of identifying and resolving errors or issues in code. For example:

>  `The team spent hours debugging the software before the release.` - Meaning they spent a long time identifying and solving these errors.

Reviewer: ```suggestion
`The team spent hours debugging the software before the release.` - Meaning they spent a long time identifying and solving these errors.
```
